copy_test.go

```diff
@@ -494,6 +483,11 @@ func TestCopy_WithOptions(t *testing.T) {
 			},
 		},
 	}
+	targetPlatform := ocispec.Platform{
+		Architecture: "test-arc-2",
+		OS:           "test-os-2",
+	}
+	opts.AddPlatformFilter(&targetPlatform)
```

Comment:
The new option function worth a standalone test function.

Reply:
I think we can have a `TestCopy_WithPlatformFilterOptions` and keep the original test case as it is.